templates/errors/no-cookie.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'layouts/_base.html' %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% import 'macros/helpers.html' as helpers %}
+
+{% set survey_title = _("Session expired") %}
+
+{% block main %}
+  <h1>{{ _("Sorry there is a problem") }}</h1>
+  <p>{{ _('To access this page you need to <a href="%(account_service_link)s">enter your 16-character access code</a> again to continue your census.', account_service_link=account_service_url) }}</p>
```

Comment:
I think the error message from the card for this ends after the link i.e. the "again to continue your census" should be removed.

Reply:
I think `account_service_url` will only be set if there is a cookie. We could introduce a fallback url if there is no cookie, but that could only go to one place which may not work for nisra.

Reply:
I checked with @donnap573 and that extra bit of the error message is correct as we're supposed to be matching the Google spreadsheet attached to the Trello card

Reply:
I had checked the Google sheet, and have just re-checked. The error text for the "Sorry, there is a problem" 401 page doesn't have the text about continuing.

Reply:
Sorry I must have checked the other page 🙈

Reply:
Yeah, I did that first!